### PR TITLE
add code  nolluda.py from feature-1

### PR DESCRIPTION
**Pull request by nolluda-johncarlo**
Please check my code if there any problem.

### CODE REVIEW

File: nolluda.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    print("==============================")
+    user_input = int(input("Enter your option: "))
+
+    if user_input == 1:
+        print("===============================================================")
+        print("Age: 18")
+        print("Birthday: September 18, 2004")
+        print("location: Guiho St. Creekland Hagonoy, Taguig City")
+        print("Hobbies: Ml, Cycling, Playing lol")
+        print("Favorites food: Bicol express, sinigang na baboy, adobong baboy ")
```
Comment (arroyo-johnmatthew) [inline nolluda.py]: "Favorites food" should be "Favorite foods"

File: nolluda.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    print("1. Basic info")
+    print("2. Goals")
+    print("3. Comments from teammates")
+    print("4. Exit")
+    print("==============================")
+    user_input = int(input("Enter your option: "))
+
+    if user_input == 1:
+        print("===============================================================")
+        print("Age: 18")
```
Comment (Lazaro-Franco) [inline nolluda.py]: please put your actual correct age

**Review by arroyo-johnmatthew — CHANGES_REQUESTED** [review]

**Review by Lazaro-Franco — CHANGES_REQUESTED** [review]
Please fix the mentioned issue

File: nolluda.py [hunk 3]
```diff
@@ -11,11 +11,11 @@
 
     if user_input == 1:
         print("===============================================================")
-        print("Age: 18")
+        print("Age: 20")
         print("Birthday: September 18, 2004")
         print("location: Guiho St. Creekland Hagonoy, Taguig City")
```
Comment (arroyo-johnmatthew) [inline nolluda.py]: location should be "Location"

**Review by arroyo-johnmatthew — CHANGES_REQUESTED** [review]

**Review by Lazaro-Franco — APPROVED** [review]
Good work! the code is good!

**Review by arroyo-johnmatthew — APPROVED** [review]
Noted. Thank you for the changes.

**Pull request merged**